pysb/core.py

```diff
@@ -1535,7 +1535,14 @@ def is_constant_expression(self):
                    for a in self.expr.atoms())
 
     def get_value(self):
-        return self.expr.evalf()
+        # Use parameter and expression values for evaluation
+        subs = {}
+        for a in self.expr.atoms():
+            if isinstance(a, Parameter):
+                subs[a] = a.value
+            elif isinstance(a, Expression) and a.is_constant_expression():
+                subs[a] = a.get_value()
+        return self.expr.evalf(subs=subs)
```

Comment:
Suggest `xreplace(subs)` instead of `evalf(subs=subs)` here. It implements just the replacement functionality we want, and it's much faster than `evalf`.

Reply:
Used xreplace as suggested. Thanks!